static/docs/command-reference/get.md

```diff
@@ -1,28 +1,40 @@
 # get
 
-Download or copy file or directory from the
-[remote storage](/doc/command-reference/remote) of any <abbr>DVC project</abbr>
-in a Git repository (e.g. hosted on GitHub) into the current working directory.
+Download a file or directory from any <abbr>DVC project</abbr> or Git repository
+(e.g. hosted on GitHub) into the current working directory.
 
-> Unlike `dvc import`, this command does not track the downloaded data files
-> (does not create a DVC-file).
+> Unlike `dvc import`, this command does not track the downloaded files (does
+> not create a DVC-file).
 
 ## Synopsis
 
 ```usage
 usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
+Download files or directories from DVC repository.
+Documentation: <https://man.dvc.org/get>
+
```

Comment:
We don't add these to the usage blocks in command references because it's redundant 🙂

Reply:
See https://github.com/iterative/dvc.org/pull/821#discussion_r350966417